cot-cli/src/migration_generator.rs

```diff
@@ -50,6 +50,53 @@ pub fn make_migrations(path: &Path, options: MigrationGeneratorOptions) -> anyho
     Ok(())
 }
 
+pub fn list_migrations(path: &Path) -> anyhow::Result<()> {
+    match find_cargo_toml(
+        &path
+            .canonicalize()
+            .with_context(|| "unable to canonicalize Cargo.toml path")?,
+    ) {
+        Some(cargo_toml_path) => {
+            let manifest = Manifest::from_path(&cargo_toml_path)
+                .with_context(|| "unable to read Cargo.toml")?;
+
+            let package_roots = match manifest.workspace {
```

Comment:
Can we extract workspace-handling code to a separate, reusable function? I think `migration make` should one day handle entire workspaces, too (probably out of scope for this PR, given that dependency cycles might happen). If we extract this, not only the code here will be cleaner, but also it might prove useful in the future.

Reply:
> Can we extract workspace-handling code to a separate, reusable function?

Sure, but I'll do it in a separate PR.